.github/workflows/generate-changelog.yml

```diff
@@ -25,8 +26,9 @@ jobs:
             else
               MSG="Update CHANGELOG.md for #${{ github.event.pull_request.number }}"
             fi
-            git config --local user.email changelogbot@mongodb.com
-            git config --local user.name changelogbot
+            git config --local user.email svc-api-experience-integrations-escalation@mongodb.com
+            git config --local user.name svc-apix-bot
+            git remote set-url origin https://x-access-token/:${{ secrets.APIX_BOT_PAT }}@github.com/${{ github.repository }}
```

Comment:
[q] using this example as [reference](https://stackoverflow.com/a/49007415), is x-access-token the username in this case?

Reply:
was using [this](https://github.com/orgs/community/discussions/25305#discussioncomment-5537395) as reference but I think you are right. Also it can be simplified even more, like [here](https://stackoverflow.com/questions/64076798/how-can-i-trigger-a-github-actions-workflow-on-push-to-another-branch/64078507#64078507)